src/Routes/SamplePage/SamplePage.js

```diff
@@ -1,38 +1,96 @@
 import React, { useEffect } from 'react';
 import { withRouter } from 'react-router-dom';
 
-import { Button, StackItem, Stack, Title } from '@patternfly/react-core';
+import {
+  Button,
+  StackItem,
+  Stack,
+  Title,
+  Select,
+  SelectOption,
+  SelectVariant,
+} from '@patternfly/react-core';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import {
   PageHeader,
   PageHeaderTitle,
 } from '@redhat-cloud-services/frontend-components/PageHeader';
+import axios from 'axios';
 
 import './sample-page.scss';
 import ProvisioningWizard from '../../Components/Wizard';
 
 /**
- * A smart component that handles all the api calls and data needed by the dumb components.
- * Smart components are usually classes.
- *
- * https://reactjs.org/docs/components-and-props.html
- * https://medium.com/@thejasonfile/dumb-components-and-smart-components-e7b33a698d43
+This page demonstrates the provisioning UI wizard
  */
+
+// This default AMI value is taken when no image builder api found
+const DEFAULT_AMI = 'ami-05fa00d4c63e32376';
```

Comment:
problem with preset ami is, that it has to be shared with the account you're about to deploy into, so this image is deployable only to your account if I'm not mistaken. so allowing to fill out the ami manually would be quite awesome for QE, without it everyone would have to use your account to test with in ephemeral.

Reply:
This `ami` is just example. it's not mine :)

Do we actually want QE test this demo page? it's not part of the production
but we can add an hidden input so QE can edit it during tests

Reply:
not just QE, but anyone in the ephemeral environment.
I mean without adding any input, QE (nor devs) can trigger the real deployment, because there is no other way to provide an image.